src/SFA.DAS.AssessorService.Application/Handlers/Login/LoginHandler.cs

```diff
@@ -118,8 +116,6 @@ public async Task<LoginResponse> Handle(LoginRequest request, CancellationToken
                 }
             }
 
-            _logger.LogInformation(LoggingConstants.SignInSuccessful);
```

Comment:
Assuming this is now redundant, although isn't it useful knowing it was successful in the logs?

Reply:
It's actually repeated above - you can't see it in the diff view - but the result would have been Successful Successful in the log - so I removed the duplicate.